py3status/modules/online_status.py

```diff
@@ -21,13 +20,7 @@
 @author obb
 """
 
-import os
-import subprocess
-try:
-    # python3
-    from urllib.request import urlopen
-except:
-    from urllib2 import urlopen
+from urllib.parse import urlsplit
```

Comment:
what happens if you run `py3status2 -l <logfile> -c <config>` on cli?  anything in the log?Also, I 

> think I'm nearly out of modules to optimize too. 😓

I'm sure there is always more, anyhow we should try to finish you outstanding PRs